src/wallet/api/schemas/walletApi.ts

```diff
@@ -0,0 +1,513 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { z } from 'zod';
+import { bigIntCoercibleSchema } from '../../../utils/bigint';
+
+/**
+ * Base response schema that all API responses extend from.
+ * Contains a success flag indicating if the operation was successful.
+ */
+const baseResponseSchema = z.object({
+  success: z.boolean(),
+});
+
+/**
+ * Schema for individual address information.
+ * Represents a single address in the wallet with its derivation index and transaction count.
+ */
+export const getAddressesObjectSchema = z.object({
+  address: z.string(), // Address in base58
```

Comment:
We can use a more specific address schema ([reference](https://github.com/HathorNetwork/hathor-wallet-lib/blob/fix/big-int-http-requests/src/template/transaction/instructions.ts#L71))

Reply:
Done! Thanks!